ui/src/Components/EditProfile.jsx

```diff
@@ -0,0 +1,266 @@
+import React from 'react';
+import { Button, Col, Form, FormGroup, Label, Input} from 'reactstrap';
+import NavBar from './NavBar';
+
+//Uncomment/Comment based on env
+// const URL = "http://localhost:3001/dev/editProfile";
+const URL = "https://bixe448nsa.execute-api.us-west-1.amazonaws.com/dev/editProfile";
+
+class CreateProfile extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            loading: true,
+            profile: null,
+            props: props,
+            firstName: '',
+            lastName: '',
+            email: '',
+            number: '',
+            profile_id: '',
+            ref1Name: '',
+            ref1Email: '',
+            ref2Name: '',
+            ref2Email: '',
+            govIdFlag: false,
+            govId: null,
+            toolPicFlag: false,
+            toolPic: null,
+        };
+
+        this.handleFirstNameChange = this.handleFirstNameChange.bind(this);
+        this.handleLastNameChange = this.handleLastNameChange.bind(this);
+        this.handleEmailChange = this.handleEmailChange.bind(this);
+        this.handleNumChange = this.handleNumChange.bind(this);
+        this.handleRef1NameChange = this.handleRef1NameChange.bind(this);
+        this.handleRef1EmailChange = this.handleRef1EmailChange.bind(this);
+        this.handleRef2NameChange = this.handleRef2NameChange.bind(this);
+        this.handleRef1EmailChange = this.handleRef2EmailChange.bind(this);
+        this.handleLicenseUpload = this.handleLicenseUpload.bind(this);
+        this.handleToolUpload = this.handleToolUpload.bind(this);
+        this.handleSubmit = this.handleSubmit.bind(this);
+
+    }
+
+    // function to get new information 
+    async componentDidMount()
+    {
+      const url = `https://bixe448nsa.execute-api.us-west-1.amazonaws.com/dev/viewProfile?id=${this.state.props.match.params.id}`;
```

Comment:
We should declare this at the top of the file like we do for the first URL to allow for easy switching back and forth bw localhost and aws. Then over here we should just concatenate the `this.state.props.match.params.id`.